.gitignore

```diff
@@ -35,3 +35,5 @@ celerybeat-schedule
 
 # Ignore coverage results
 coverage
+coverage.xml 
+.coverage
```

Comment:
This comment is not for this file but the one above:
<img width="217" alt="image" src="https://user-images.githubusercontent.com/44410/73546388-2556dc80-440b-11ea-9396-5ac9f7700a3c.png">

Could you please remove the file `.coverage`?

A good practice for you to have is to review your own PR before asking for review. That might have helped you noticed that such file is part of your PR.

Reply:
Really sorry, sir. Will keep this in mind